libs/examples/src/table/table-empty-state-example/table-empty-state-example.html

```diff
@@ -23,16 +23,16 @@
         />
       </dt-empty-state-item-img>
 
-      <dt-empty-state-item-title>
+      <dt-empty-state-item-title aria-level="1">
```

Comment:
Same as I mentioned above. Do we also have to add `role="heading"`? And why is this one a level 1 heading?